riscv/src/compiler.rs

```diff
@@ -690,10 +690,46 @@ fn runtime(coprocessors: &CoProcessors) -> String {
 .globl __udivdi3
 .set __udivdi3@plt, __udivdi3
 
+.globl __udivti3@plt
+.globl __udivti3
+.set __udivti3@plt, __udivti3
+
+.globl __divdf3@plt
+.globl __divdf3
+.set __divdf3@plt, __divdf3
+
+.globl __muldf3@plt
```

Comment:
Can you use something like
`["udivdi3", "__udivti3", "__divdf3", ...].map(|n| format!(".globl {n}@plt\n.globl {n}\n.set {n}@plt, {n}\n")).join("\n\n")`?